admin/create-theme/theme-utils.php

```diff
@@ -208,4 +208,17 @@ public static function replace_screenshot( $new_screenshot_path ) {
 		return CBT_Theme_Utils::copy_screenshot( $new_screenshot_path );
 	}
 
+	/**
+	 * Get the current WordPress version.
+	 *
+	 * @return string The current WordPress in the format x.x (major.minor)
+	 * Example: 6.5
+	 */
+	public static function get_current_wordpress_version() {
+		$wp_version = get_bloginfo( 'version' );
+		if ( preg_match( '/^\d+\.\d+/', $wp_version, $matches ) ) {
```

Comment:
Why is this regex necessary? Is the version ever not in `major.minor` or `major.minor.patch` format? 

I think I'd favor using explode instead, to avoid directly using regex.

Reply:
I'm impartial to how the version is snagged.  The regex fairy gave me that and it worked.  I wouldn't be sad if you refactored it.